### PR TITLE
[hue] Removed internal vendor Map and minor additional improvements

### PR DESCRIPTION
**Pull request by cweitkamp**
- Removed internal vendor Map as the vendor is exposed by the API. We additionally not only support Philips (or Signify Netherlands B.V.) and OSRAM but a lot others too
- Minor improvements of console command messages
- Added "members" property for groups
- Added German translation for scene Channel

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueGroupHandler.java [hunk 1]
```diff
@@ -123,6 +127,15 @@ private void initializeThing(@Nullable ThingStatus bridgeStatus) {
         }
     }
 
+    private synchronized void initializeProperties(@Nullable FullGroup fullGroup) {
+        if (!propertiesInitializedSuccessfully && fullGroup != null) {
```
Comment (lolodomo) [inline bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueGroupHandler.java:131]: The members of a group can be changed (form the Hue app). In this case, the property will not be updated.
What is exactly the purpose of this new property in openHAB ?

Comment (cweitkamp) [reply bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueGroupHandler.java:131]: You are right. We do not need this flag. The framework checks for changes.

The property itself shows which members are part of the group.

**Review by lolodomo — APPROVED** [review]
LGTM

**Review by cpmeister — APPROVED** [review]
LGTM

**Pull request merged**